staging/src/k8s.io/client-go/util/workqueue/queue_test.go

```diff
@@ -321,3 +324,44 @@ func TestQueueDrainageUsingShutDownWithDrainWithDirtyItem(t *testing.T) {
 
 	finishedWG.Wait()
 }
+
+
+
```

Comment:
drop the "Created outside" comment and run hack/update-gofmt.sh to clean up formatting, then lgtm